scripts/run-docker.sh

```diff
@@ -15,14 +17,17 @@ docker start $CONTAINER_NAME > /dev/null 2> /dev/null || {
 	       --detach \
 	       --env HOME=$HOME \
 	       --name $CONTAINER_NAME \
-	       --volume $PWD:$HOME/termux-packages \
+	       --volume $REPOROOT:$HOME/termux-packages \
 	       --tty \
 	       $IMAGE_NAME
-
-	echo "Changed builder uid/gid..."
-	docker exec $CONTAINER_NAME chown -R `id -u` /data >& /dev/null
-	docker exec $CONTAINER_NAME usermod -u `id -u` builder >& /dev/null
-	docker exec $CONTAINER_NAME groupmod -g `id -g` builder >& /dev/null
+	if [ $(id -u) -ne 1000 ]
```

Comment:
Or wait, this changes the ownership of everything to root:root if run-docker.sh is run as root. This clearly is undesired (it pretty much creates the same error as you wanted to avoid, packages are build as root and then aren't accessable for the normal user. In fact, even the .git folder is chown'd to root so you pretty much do nothing as normal user). 

I think if `[ $(id -u) -ne 1000 ]` should be replaced by `if [ $(id -u) -ne 1000 -a $(id -u) -ne 0 ]` to avoid this.